hack/run-e2e-test

```diff
@@ -16,151 +16,190 @@
 
 set -euo pipefail
 
-OS_ARCH=$(go env GOOS)-amd64
-TEST_ID=$RANDOM
-CLUSTER_NAME=test-cluster-$TEST_ID
-TEST_DIR=/tmp/ebs-e2e-test
-BASE_DIR=$(dirname $0)
-REGION=${AWS_REGION-us-west-2}
-ZONES=${AWS_AVAILABILITY_ZONES-us-west-2a,us-west-2b,us-west-2c}
-FOCUS=${GINKGO_FOCUS-"[ebs-csi-e2e]"}
-NODES=${GINKGO_NODES:-4}
-K8S_VERSION=${K8S_VERSION-1.18.10}
-INSTANCE_TYPE=${INSTANCE_TYPE-c4.large}
-
-source $(dirname "${BASH_SOURCE}")/utils/helm.sh
-
-echo "Testing in region: $REGION and zones: $ZONES"
+TEST_ID=${TEST_ID:-$RANDOM}
+CLUSTER_NAME=test-cluster-${TEST_ID}
 
-KOPS_DOWNLOAD_URL=https://github.com/kubernetes/kops/releases/download/v1.18.2/kops-$OS_ARCH
-KOPS_PATH=$TEST_DIR/kops
-KOPS_STATE_FILE=s3://k8s-kops-csi-e2e
+BASE=$(realpath "${BASH_SOURCE[0]}")
+BASE_DIR=$(dirname "${BASE}")
+TEST_DIR=${BASE_DIR}/ebs-e2e-test
+BIN_DIR=${TEST_DIR}/bin
+SSH_KEY_PATH=${TEST_DIR}/id_rsa
 
-# Download kops if not yet
-if [[ ! -e $KOPS_PATH ]]; then
-    mkdir -p $TEST_DIR
-    echo "Downloading KOPS from $KOPS_DOWNLOAD_URL to $KOPS_PATH"
-    curl -L -X GET $KOPS_DOWNLOAD_URL -o $KOPS_PATH
+REGION=${AWS_REGION-us-west-2}
+ZONES=${AWS_AVAILABILITY_ZONES:-us-west-2a,us-west-2b,us-west-2c}
+INSTANCE_TYPE=${INSTANCE_TYPE:-c4.large}
+
+K8S_VERSION=${K8S_VERSION:-1.18.10}
+KOPS_VERSION=${KOPS_VERSION:-1.18.2}
+KOPS_STATE_FILE=${KOPS_STATE_FILE:-s3://k8s-kops-csi-e2e}
+
+KUBECONFIG=${KUBECONFIG:-"${HOME}/.kube/config"}
+ARTIFACTS=${ARTIFACTS:-"${TEST_DIR}/artifacts"}
+GINKGO_FOCUS=${GINKGO_FOCUS:-"\[ebs-csi-migration\]"}
```

Comment:
Just to verify, we're changing from `ebs-csi-e2e` to `ebs-csi-migration`?

Reply:
yeah, just for the default. The non-migration tests currently go through the tester binary instead of this script anyway. (the theory behind the tester binary is that we can update it once to update all 3 aws csi drivers, so we don't need to maintain 3 almost-identical kops "run-e2e-test" scripts in all 3 repos, but in practice we need to do that anyway since the binary configuratoin yaml is effectively a script: https://github.com/kubernetes-sigs/aws-ebs-csi-driver/blob/master/tester/single-az-config.yaml)